Gemfile

```diff
@@ -9,7 +9,7 @@ gem "sentry-rails"
 gem 'scout_apm'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
-gem 'rails', '~> 6.1.3', '>= 6.1.3.2'
```

Comment:
why?

Reply:
> why?

There was problem with Stack Level Error, because I rendered representer, even though it wasnt needed in post request buy method. So before I realized that was it, I searched stackoverflow to find solution and there was this hint, but this update, doesnt do anything bad in project.